app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if local_assigns[:model].errors.any? %>
+  <div class="alert alert-danger" role="alert">
+    <h2 class="alert__heading">There's a problem</h2>
+    <ul>
+      <% local_assigns[:model].errors.full_messages.each do |msg| %>
```

Comment:
This call and the one one line 1 will error out with an unuseful error message if model is nil.